packages/admin/cms-admin/src/generator/future/generateForm/generateFormLayout.ts

```diff
@@ -0,0 +1,53 @@
+import { FormConfig, FormLayoutConfig } from "../generator";
+import { Imports } from "../utils/generateImportsCode";
+
+export function generateFormLayout(
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    config: FormLayoutConfig<any>,
+    fieldsCode: string,
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    formConfig: FormConfig<any>,
+) {
+    const gqlType = formConfig.gqlType;
+    const instanceGqlType = gqlType[0].toLowerCase() + gqlType.substring(1);
+
+    const imports: Imports = [];
+    let code = "";
+    const formFragmentFields: string[] = [];
+
+    if (config.type === "fieldSet") {
+        imports.push({ name: "FieldSet", importPath: "@comet/admin" });
+        const supportPlaceholder = config.supportText?.includes("{");
+        if (supportPlaceholder) {
+            imports.push({ name: "FormSpy", importPath: "react-final-form" });
+        }
+        code = `
+        <FieldSet
+            collapsible={${String(config.collapsible)}}
+            initiallyExpanded={${String(config.initiallyExpanded)}}
```

Comment:
I think you should better generate
```suggestion
            ${config.initiallyExpanded ? "initiallyExpanded" : ""}
```